handlers/changes.js

```diff
@@ -161,15 +169,21 @@ var cleanUp = function(feed) {
 };
 
 var getChanges = function(feed) {
+  var allIds = _.union(feed.requestedIds, feed.validatedIds);
+  var chunks = [];
+  while (allIds.length) {
+    chunks.push(allIds.splice(0, MAX_DOC_IDS));
+  }
+
   // we cannot call 'changes' in nano because it only uses GET requests and
   // our query string might be too long for GET
-  feed.changesReq = db.request({
+  async.parallel(chunks.map(docIds => (callback) => db.request({
```

Comment:
Thinking we might want to not do parallel here, so that we stress the server less? I guess it really depends on what that 100 limit is guarding against.